### PR TITLE
Enable viewers to post comments and show all comments

### PR DESCRIPTION
**Pull request by KathleenQ**
The changes added a comment box to enable viewers to post comments. All comments will be stored in a list and transferred to JSON string to show them.

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -23,6 +23,12 @@
       <version>4.0.1</version>
       <scope>provided</scope>
     </dependency>
+
```
Comment (lilian-hunt) [inline portfolio/pom.xml:26]: remove this space, have you run the clang-format ?

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -24,9 +27,21 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private List<String> commentsStorage = new ArrayList<>();
```
Comment (lilian-hunt) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:29]: run clang format on this for spacing too

Comment (KathleenQ) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:29]: Yes, I have just adjusted the format again according to clang format. Thank you for pointing it out!

File: portfolio/pom.xml (hunk 1 above)
Comment (KathleenQ) [reply portfolio/pom.xml:26]: I don't think this space needs to be removed, according to clang-format result. But thank you for viewing that! I have just adjusted all files in step/portfolio again. :) And I have made changes for the space you pointed below.

**Review by aannie-y — COMMENTED** [review]
looks good to me

**Review by tripd22 — APPROVED** [review]
LGTM

**Pull request merged**